zebra-network/src/config.rs

```diff
@@ -597,6 +597,7 @@ impl<'de> Deserialize<'de> for Config {
             activation_heights: Option<ConfiguredActivationHeights>,
             pre_nu6_funding_streams: Option<ConfiguredFundingStreams>,
             post_nu6_funding_streams: Option<ConfiguredFundingStreams>,
+            halving_interval: Option<u32>,
```

Comment:
Could we rename this to `pre_blossom_halving_interval`? The post-Blossom halving interval should be double the pre-Blossom interval, and they're integers.